lib/ansible/modules/windows/setup.ps1

```diff
@@ -439,6 +439,43 @@ if($gather_subset.Contains('winrm')) {
     }
 }
 
+if($gather_subset.Contains('virtual')) {
+    $MachineInfo = Get-WmiObject -Class Win32_ComputerSystem
+
+        switch ($MachineInfo.Model) {
```

Comment:
Something wrong with the indentation here ?

Reply:
Updated the source, and will resubmit

Reply:
Removed executable flag from the file, and also have changed Get-WMIObject to Get-CimInstance